tutorial.md

```diff
@@ -116,8 +118,11 @@ sudo apt-get update && apt-get install -y libglib2.0-dev \
                                           libseccomp-dev \
                                           libgpgme11-dev \
                                           libdevmapper-dev \
-                                          make \
-                                          git
+					  libudev-dev \
```

Comment:
Do we need to add another space?

Reply:
I just noticed it looks like a missing space in the e-mail about the PR comment as well but on it looks fine to me when I preview the markdown page

Reply:
> I just noticed it looks like a missing space in the e-mail about the PR comment as well but on it looks fine to me when I preview the markdown page

You have tab indents here, while the previous lines have space indents:

```console
$ sed 's/\t/[TAB]/g' tutorial.md | grep -3 libudev-dev
                                          libseccomp-dev \
                                          libgpgme11-dev \
                                          libdevmapper-dev \
[TAB][TAB][TAB][TAB][TAB]  libudev-dev \
[TAB][TAB][TAB][TAB][TAB]  go-md2man \
[TAB][TAB][TAB][TAB][TAB]  software-properties-common \
[TAB][TAB][TAB][TAB][TAB]  gcc \
```

I recommend switching to spaces.